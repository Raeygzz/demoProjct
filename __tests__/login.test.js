import 'react-native';
import React from 'react';
import Login from '../src/screens/login/login';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Login />).toJSON();
  expect(tree).toMatchSnapshot();
});
