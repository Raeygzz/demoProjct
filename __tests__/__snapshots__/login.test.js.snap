// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
        "justifyContent": "center",
      },
      Object {
        "borderTopLeftRadius": 24,
        "borderTopRightRadius": 24,
      },
      Object {
        "backgroundColor": "#171635",
      },
      Object {
        "paddingBottom": 34,
        "paddingHorizontal": 27,
        "paddingTop": 50,
      },
    ]
  }
>
  <RCTScrollView
    contentContainerStyle={
      Object {
        "flexGrow": 1,
        "justifyContent": "center",
      }
    }
    keyboardShouldPersistTaps="handled"
  >
    <View>
      <View
        style={
          Array [
            Object {
              "display": "flex",
              "justifyContent": "flex-end",
              "marginBottom": 20,
              "minHeight": 68,
              "overflow": "hidden",
              "paddingTop": 20,
              "position": "relative",
            },
            undefined,
            Object {
              "paddingBottom": 0,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontFamily": "Apercu_Regular",
              "fontSize": 14,
              "left": 0,
              "lineHeight": 17,
              "opacity": 0.68,
              "position": "absolute",
              "top": 26,
            }
          }
        >
          Email Address
        </Text>
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          autoFocus={true}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={[Function]}
          placeholderTextColor="rgba(255,255,255,0.7)"
          rejectResponderTermination={true}
          returnKeyType="next"
          selectionColor="#71648E"
          style={
            Array [
              Object {
                "borderBottomColor": "white",
                "borderBottomWidth": 1,
                "color": "white",
                "fontFamily": "Apercu Medium",
                "fontSize": 18,
                "padding": 0,
              },
              null,
              undefined,
              Object {
                "borderBottomColor": "white",
                "height": 37,
                "lineHeight": 12,
              },
              null,
            ]
          }
          underlineColorAndroid="transparent"
          value=""
        />
      </View>
      <View
        style={
          Array [
            Object {
              "display": "flex",
              "justifyContent": "flex-end",
              "marginBottom": 20,
              "minHeight": 68,
              "overflow": "hidden",
              "paddingTop": 20,
              "position": "relative",
            },
            undefined,
            Object {
              "paddingBottom": 0,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontFamily": "Apercu_Regular",
              "fontSize": 14,
              "left": 0,
              "lineHeight": 17,
              "opacity": 0.68,
              "position": "absolute",
              "top": 26,
            }
          }
        >
          Password
        </Text>
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onSubmitEditing={null}
          placeholderTextColor="rgba(255,255,255,0.7)"
          rejectResponderTermination={true}
          returnKeyType="done"
          secureTextEntry={true}
          selectionColor="#71648E"
          style={
            Array [
              Object {
                "borderBottomColor": "white",
                "borderBottomWidth": 1,
                "color": "white",
                "fontFamily": "Apercu Medium",
                "fontSize": 18,
                "padding": 0,
              },
              null,
              undefined,
              Object {
                "borderBottomColor": "white",
                "height": 37,
                "lineHeight": 12,
              },
              null,
            ]
          }
          underlineColorAndroid="transparent"
          value=""
        />
      </View>
      <View
        style={
          Object {
            "height": 50,
          }
        }
      >
        <Text>
          Â 
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#ffbcbc",
              "borderColor": "red",
              "borderWidth": 1,
              "height": 100,
              "justifyContent": "center",
              "margin": 10,
              "width": 300,
            },
            Object {
              "width": "100%",
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "#333333",
              "margin": 20,
            }
          }
        >
          TouchableNativeFeedback is not supported on this platform!
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
